### PR TITLE
Updates F02iH tutorial - nft (uplift to ink! v4)

### PR DESCRIPTION
**Pull request by Maar-io**
Updated code and text for NFT tutorial related to ink! v4

### CODE REVIEW

File: docs/build/wasm/from-zero-to-ink-hero/nft/CustomTrait/customtrait.md [hunk 1]
```diff
@@ -36,26 +36,21 @@ pub mod impls;
...
 
 scale = { package = "parity-scale-codec", version = "3", default-features = false, features = ["derive"] }
-scale-info = { version = "2", default-features = false, features = ["derive"], optional = true }
+scale-info = { version = "2.3", default-features = false, features = ["derive"], optional = true }
 
-openbrush = { tag = "v2.3.0", git = "https://github.com/Supercolony-net/openbrush-contracts", default-features = false, features = ["ownable", "psp34", "reentrancy_guard"] }
+openbrush = { tag = "3.0.0", git = "https://github.com/727-Ventures/openbrush-contracts", default-features = false, features = ["psp34", "ownable"] }
 
 [lib]
 path = "lib.rs"
```
Comment (ashutoshvarma) [inline docs/build/wasm/from-zero-to-ink-hero/nft/CustomTrait/customtrait.md:56]: no need for `crate-type` - https://use.ink/faq/migrating-from-ink-3-to-4/#removal-of-libcrate-type-and-libname-from-contract-manifest

File: docs/build/wasm/from-zero-to-ink-hero/nft/Wizard/wizard.md [hunk 2]
```diff
@@ -76,25 +79,17 @@ Your `Cargo.toml` should now look like this:
...
 scale = { package = "parity-scale-codec", version = "3", default-features = false, features = ["derive"] }
-scale-info = { version = "2", default-features = false, features = ["derive"], optional = true }
+scale-info = { version = "2.3", default-features = false, features = ["derive"], optional = true }
 
-# Include brush as a dependency and enable default implementation for PSP34 via brush feature
-openbrush = { tag = "v2.3.0", git = "https://github.com/Supercolony-net/openbrush-contracts", default-features = false, features = ["psp34", "ownable"] }
+openbrush = { tag = "3.0.0", git = "https://github.com/727-Ventures/openbrush-contracts", default-features = false, features = ["psp34", "ownable"] }
 
 [lib]
 name = "shiden34"
```
Comment (ashutoshvarma) [inline docs/build/wasm/from-zero-to-ink-hero/nft/Wizard/wizard.md:95]: same, https://use.ink/faq/migrating-from-ink-3-to-4/#removal-of-libcrate-type-and-libname-from-contract-manifest

File: docs/build/wasm/from-zero-to-ink-hero/nft/CustomTrait/customtrait.md (hunk 1 above)
Comment (Maar-io) [reply docs/build/wasm/from-zero-to-ink-hero/nft/CustomTrait/customtrait.md:56]: ok

File: docs/build/wasm/from-zero-to-ink-hero/nft/Wizard/wizard.md (hunk 2 above)
Comment (Maar-io) [reply docs/build/wasm/from-zero-to-ink-hero/nft/Wizard/wizard.md:95]: ok

**Review by ashutoshvarma — APPROVED** [review]

**Comment by andabak** [comment]
LGTM (from Docusaurus point of view)

**Review by andabak — APPROVED** [review]

**Review by fiexer — APPROVED** [review]

**Pull request merged**